classes/class-woothemes-sensei-certificates.php

```diff
@@ -399,9 +399,9 @@ public function post_type_custom_column_content( $column_name, $post_ID ) {
 	 * @param  int $course_id data to post
 	 * @return void
 	 */
-	public function generate_certificate_number( $user_id = 0, $course_id = 0 ) {
+	public function generate_certificate_number( $status, $user_id = 0, $course_id = 0 ) {
```

Comment:
This function is defined as `public` access (although it probably shouldn't have been). We'll need to handle this since we're changing the signature.

Reply:
Good call. I reverted this function back and added an `@access private` wrapper function instead. abe5833